### PR TITLE
Add Option Class

### LINKED ISSUE #15: Implement options in quiz

### PR DESCRIPTION
**Pull request by anqi20**
Fixes #15

### CODE REVIEW

File: src/main/java/seedu/duke/option/OptionInterface.java [hunk 1]
```diff
@@ -1,8 +1,9 @@
-package seedu.duke;
+package seedu.duke.option;
 
 public interface OptionInterface {
     String getOptionDescription();
 
     boolean isCorrectAnswer();
 
+    void markAsCorrectAnswer();
```
Comment (kstonekuan) [inline src/main/java/seedu/duke/option/OptionInterface.java:8]: Good catch, forgot to include this.

File: src/main/java/seedu/duke/option/OptionList.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package seedu.duke.option;
+
+import java.util.ArrayList;
+
+public class OptionList implements OptionListInterface {
+    private final ArrayList<OptionInterface> optionList;
```
Comment (kstonekuan) [inline src/main/java/seedu/duke/option/OptionList.java]: I think this one may have to be "options" as it is a collection. When we call an instance of the OptionList we can probably call that optionList

File: src/main/java/seedu/duke/option/OptionList.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private int optionCount;
```
Comment (kstonekuan) [inline src/main/java/seedu/duke/option/OptionList.java]: Good idea but may be redundant as we can call "length()" on the ArrayList

**Review by kstonekuan — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/option/OptionList.java (hunk 3 above)
Comment (josephhhhhhhhh) [reply src/main/java/seedu/duke/option/OptionList.java]: Ah yes, this makes sense. We can cut down on unnecessary variables this way!

File: src/main/java/seedu/duke/option/OptionList.java (hunk 2 above)
Comment (anqi20) [reply src/main/java/seedu/duke/option/OptionList.java]: Ok, I will take note of this and change them. Thanks for the explanation!

File: src/main/java/seedu/duke/option/OptionInterface.java (hunk 1 above)
Comment (anqi20) [reply src/main/java/seedu/duke/option/OptionInterface.java:8]: 👍

File: src/main/java/seedu/duke/option/OptionList.java (hunk 3 above)
Comment (anqi20) [reply src/main/java/seedu/duke/option/OptionList.java]: Ahh, slipped my mind! Thank you for pointing it out!

**Review by kstonekuan — APPROVED** [review]
LGTM, good work

**Pull request merged**